README.md

```diff
@@ -1,24 +1,271 @@
-application up and running.
 
-Things you may want to cover:
+<!-- TABLE OF CONTENTS -->
 
-* Ruby version
+# 📗 Table of Contents
 
-* System dependencies
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
 
-* Configuration
 
-* Database creation
 
-* Database initialization
 
-* How to run the test suite
 
-* Services (job queues, cache servers, search engines, etc.)
 
-* Deployment instructions
+<!-- PROJECT DESCRIPTION -->
 
-* ...
+# 📖 [Spend Right Budget App] <a name="about-project"></a>
+
+**Spend Right** is a full stack ruby on rails web application on a postgreSQL database that is built to manage your finances by grouping your expenses or transactions into categories and helping you visualize how much your are spending and where you are spending the most amount of money.
+
+## 🛠 Built With <a name="built-with">Ruby on Rails</a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>HTML</li>
+    <li>CSS</li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://render.com/">Render</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+
+
+
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+- **Spash screen to intoduce user to app**
+- **Total amount of transactions in each category**
+- **Ability to add new category**
+- **Ability to add new transaction to a category**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+
+
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- [Live Demo Link](https://spend-right.onrender.com)
+
+![image](https://user-images.githubusercontent.com/107795045/221205973-786b7a1f-0300-4f87-8cc8-133d1ac666f9.png) ![image](https://user-images.githubusercontent.com/107795045/221206259-97d0af93-95d0-4142-8989-b3e51b25b7d3.png) ![image](https://user-images.githubusercontent.com/107795045/221206553-d6de17ad-da18-49bd-89ae-2c3fc31b5cfc.png)
+![image](https://user-images.githubusercontent.com/107795045/221207104-b5ec97c4-5882-4b74-8d9e-f9edd19691c4.png) ![image](https://user-images.githubusercontent.com/107795045/221207526-6ab186bd-902b-4300-bb23-510955fa254d.png)
```

Comment:
- [ ] There should be a video presenting the features of the project.
    - The maximum length of the video is 5 minutes.
    - The video contains a clear and objective description about the project in professional English. The topic and purpose of the project are made clear.
    - The video contains a demo of the project, highlighting all the project features.
    - The video contains a highlight about a piece of the code the student finds interesting or is proud of making.

Reply:
An unpardonable oversight 😅. I will add that as soon as possible